lib/neo4j/active_node/query/query_proxy.rb

```diff
@@ -258,6 +254,7 @@ def new_link(node_var = nil)
           self.clone.tap do |new_query_proxy|
             new_query_proxy.instance_variable_set('@result_cache', nil)
             new_query_proxy.instance_variable_set('@node_var', node_var) if node_var
+            new_query_proxy.instance_variable_set('@distinct', @distinct) if @distinct
```

Comment:
Good catch, I think it was a bit of code from something that I was playing with.  It wasn't something that was complete so I just removed from the `8.1.x` branch